tarsier/__main__.py

```diff
@@ -6,14 +6,21 @@
 from playwright.async_api import async_playwright
 
 from tarsier.core import Tarsier
-from tarsier.ocr import GoogleVisionOCRService
+from tarsier.ocr import GoogleVisionOCRService, MicrosoftAzureOCRService
 
 
-async def main(credentials_path: str, url: str, verbose: bool) -> None:
+
+async def main(credentials_path: str, url: str, verbose: bool, ocr_service: str = "google") -> None:
```

Comment:
instead of having ocr_service be type str we should probably make a type:

OcrProvider = Literal["google", "microsoft"]

Reply:
Sure, did it slightly differently using StrEnum. Defined in tarsier/ocr